cedar-policy-core/src/ast/entity.rs

```diff
@@ -346,6 +355,19 @@ impl Entity {
         })
     }
 
+    /// Creates a new `Entity` with this UID, ancestors, and an empty set of attributes
```

Comment:
Consistency with `new` doc comment.
```suggestion
    /// Create a new `Entity` with this UID, ancestors, and an empty set of attributes
```